src/Interface/InterfaceView/InterfaceView.js

```diff
@@ -133,9 +155,14 @@ export const InterfaceView = ({ item = {}, isNarrow = false }) => {
   );
 };
 
+InterfaceView.manifest = {
```

Comment:
I think you should have  `InterfaceViewContainer` for this

Reply:
done